### ISSUE TITLE
The layout is disrupted by the boost extension on Edge app. 

### ISSUE THREAD

**Issue by RENSHUZHE** [GorvGoyl/Notion-Boost-browser-extension#58]
So here is the issue, Microsoft edge has a function called "apps" which enable the users to download a web page as an web  app on desktop for easier access. I download the Notion.so page as the web app and I also used the notion boost extension. However, recently, I found that the layout of my notion page has been disrupted like this
![image](https://user-images.githubusercontent.com/49580743/135200026-dc37bffe-4a88-44f0-a306-6e281230d8bc.png)
I have tried elimination method and found out that it is notion boost causing this problem. If I opened the page on the edge instead of on the app, the layout is just as fine as usual with notion boost. So I guess it is because the notion boost has some incompatibility with edge app?  I have tried delete the app and redownload the web page. However, this problem still exists.

**Comment by RENSHUZHE** [comment]
Sorry,  think the issue is not just related to the app function, but to the edge developer edition (sorry for the confusion, I forgot that I was not using the usual edge browser). I opened the notion page on the edge dev browser, at first it appears okay, but then if I copied the link and opened it in a new tab, these things will happen again. 

So I have found a way around it, the usual edge do not have this problem. So I think this is related to some new updated features in Edge dev.
